alex/components/dm/base.py

```diff
@@ -214,14 +208,3 @@ def log_state(self):
         :return: none
         """
         self.dialogue_state.log_state()
-
-    def get_token(self):
```

Comment:
OK!

Best regards,
Filip Jurcicek

-------------------------------------------------------
Work tel. (CZ): +420221914402
Personal tel. (CZ): +420777805048
Skype: bozskyfilip

http://ufal.mff.cuni.cz/filip-jurcicek
http://ufal.mff.cuni.cz/grants


On 27 February 2015 at 10:10, Lukas Zilka <notifications@github.com> wrote:

> In alex/components/dm/base.py
> <https://github.com/UFAL-DSG/alex/pull/126#discussion_r25494653>:
>
> > @@ -214,14 +208,3 @@ def log_state(self):
> >          :return: none
> >          """
> >          self.dialogue_state.log_state()
> > -
> > -    def get_token(self):
>
> OK. I moved it to utils/token.py because I think this is a better place
> for such method as it is not a general aspect of a dialog manager.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UFAL-DSG/alex/pull/126/files#r25494653>.
>